docs/source/science/NISAR/OPERA Sentinel-1 .ipynb

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
<p>What's the benefit to using this over just the regular earthaccess lib?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>for search, none - it just hasn't historically been the MAAP way (need to check if earthaccess is installed by default now, I think it is) </p><p> </p><p>+1 we should probably switch to EarthAccess for this, particulary because we're not downloading</p>